### PR TITLE
constrain norm after learning rate scaling

### PR DESCRIPTION
**Pull request by nilin**
that way constrain_norm doesn't need to know the learning rate

### CODE REVIEW

File: vmcnet/updates/proxsr.py [hunk 1]
```diff
@@ -73,12 +73,12 @@ def proxsr_update_fn(
 
 def constrain_norm(
     grad: P,
-    learning_rate: chex.Numeric,
     norm_constraint: chex.Numeric = 0.001,
+    learning_rate_correction: chex.Numeric = 1.0,
```
Comment (ggoldsh) [inline vmcnet/updates/proxsr.py:77]: Param seems a bit oddly named given it doesn't really have to do with the learning rate any more. Could just remove it for now since it looks unused?

**Review by ggoldsh — APPROVED** [review]
LGTM mod one comment

**Comment by ggoldsh** [comment]
Actually I have a question about this. Will this result in the "prev_grad" in the next iteration being the grad BEFORE the scaling or AFTER the scaling? I guess it depends on whether it's saved in .update() or.apply_updates().

Can you check on this to make sure it's doing the right thing before merging?
